### ISSUE TITLE
Ethereum Cat Herders Meeting #78

### ISSUE THREAD

**Issue by poojaranjan** [ethereum-cat-herders/PM#268]
## Date and Time

Feb 15, 2022, at 15:00 UTC

## Location 

Zoom: https://us02web.zoom.us/j/84834303064?pwd=YTVDNUxvSjEzL1VTOXNoNmI1a2RlZz09

Video at YouTube: https://www.youtube.com/channel/UCD9iiIwTRtLDYcEWONs2Q3A

## Agenda

New users on ECH Discord and/or contributor's introduction

### 1. Ethereum network upgrades
- [The Merge](https://ethereumcatherders.com/the_merge/) 
  - Merge Community meeting 3 - [Recording](https://www.youtube.com/watch?v=65Pt6oS3kDM),  [Notes](https://github.com/ethereum/pm/blob/74a74a9931f46030d040211004e72ab60d32fc50/Merge/CommunityCall_Meeting_03.md)
  - [Kiln Spec v1](https://hackmd.io/@n0ble/kiln-spec)
- [Shanghai EIPs](https://github.com/ethereum/pm/issues)

### 2. ECH updates
- ECH website
- [ECH engineering](https://github.com/ethereum-cat-herders/PM/issues/266) 
  - EIP bot
  - [Learn2Earn](https://github.com/ethereum-cat-herders/L2E)
- Cat blazers 
   - [ECH Podcast](https://open.spotify.com/show/7dgxKMkSyy3HWtQW7OfqXA)
   - ECH Medium to Mirror 
   - ECH ENS names Ref: [Comment](https://github.com/ethereum-cat-herders/PM/issues/268#issuecomment-1039942553)
- ECH Operations 
   - ECH New Joiners Meeting February 2022
   - Women's day celebration
- [EIP Insight](https://hackmd.io/@poojaranjan/EthereumImprovementProposalsInsight)
- [PEEPanEIP](https://www.youtube.com/playlist?list=PL4cwHXAawZxqu0PKKyMzG_3BJV_xZTi1F)  
   - Upcoming meetings can be found on [PEEPanEIP schedule](https://github.com/ethereum-cat-herders/PM/projects/2)

- [Meeting notes](https://docs.google.com/spreadsheets/d/1axM1KaKc-GCcoPbAnYTOhgLnxXsBVoO5XRjTN3CLnck/edit#gid=0) and action items for ECH 
  - [Danksharding](https://www.youtube.com/watch?v=e9oudTr5BE4) 
  - All Core Dev meeting [notes 131](https://github.com/ethereum/pm/pull/474)
  - Consensus meeting [notes 81](https://github.com/ethereum/pm/blob/df3014a74134fb51f0d8faa7d7635f0eff4025c7/Consensus-Layer-Meetings/call_081.md)
  - EIP Improvement Process meeting 49
  - EIP editors apprentice meetings 12 - [Recording](https://youtu.be/tfxJtVyylII), [Summary](https://github.com/ethereum-cat-herders/EIPIP/issues/106#issuecomment-1033204064)

  
### 3. New tasks / brainstorming
- ECH website hosting via Github
- Client meetings in podcast form
- [EL Merge Client Release Naming](https://ethereum-magicians.org/t/el-merge-client-release-naming/7928/14)

### 4. Events & hackathon
- [Ethereum Rio](https://www.ethereum.rio/) - 11-16 March 2022
- [Devconnect](https://devconnect.org/) Amsterdam, Netherlands April 2022

### 5. ECH funding
- Funding via [Gitcoin Round 12](https://gitcoin.co/grants/782/ethereum-cat-herders-community-fund), CLR & [Moloch](https://twitter.com/MolochDAO/status/1472966722453835781)

### 6. Discuss and close the ECH GitHub Issues/PRs
- [ethereum-cat-herders/PM](https://github.com/ethereum-cat-herders/PM/issues)
- [ethereum-cat-herders/funding](https://github.com/ethereum-cat-herders/funding/issues)

### 7. Review of outstanding action items from previous ECH meetings
- [Meeting 77](https://github.com/ethereum-cat-herders/PM/blob/master/meeting%20077.md)

 Anything else?
#### The date for the next meeting - March 01, 2022.

**Comment by wschwab** [comment]
I'd like to talk about the ECH's ENS addresses - they expired and I grabbed them, I also picked up `ech.eth` (which is more expensive since it's a three-character), and would like to see if everyone agrees that it should be reimbursed. There's also some other technical questions around them.

Just to be clear, I picked them up on Sunday but have not yet transferred them - I'd like to make really sure I'm sending them to the right place first, and once I've confirmed that, will probably wait until next Sunday when gas will likely be cheaper again.
